pkg/ccl/changefeedccl/cdctest/nemeses.go

```diff
@@ -409,8 +410,14 @@ func RunNemesis(
 		)
 		defer queryGen.Close()
 		const numInserts = 100
+		time := timeutil.Now()
 		for i := 0; i < numInserts; i++ {
 			query := queryGen.Generate()
+			log.Infof(ctx, "Executing query: %s", query)
+			if i != 0 {
```

Comment:
I think this would be cleaner if instead of logging the execution time for the query in the last iteration of the loop, this log moved after the `db.Exec()` and happened for every iteration of the loop.

Reply:
Good idea, done.